lab/lab2/js/part1-ajax-calls.js

```diff
@@ -103,12 +105,17 @@ var phillyBikeCrashesDataUrl = "https://raw.githubusercontent.com/CPLN692-MUSA61
   Remember to call all code within the function body. Use console.log to make sure
   that this step is completed before moving on!
 ===================== */
-
+$.ajax(phillyCrimeDataUrl).done(function(ajaxResponseValue) {
+  var crimelist = JSON.parse(ajaxResponseValue);
```

Comment:
Does this load data on to the map for you?

You need to define crimelist outside of the ajax call so it can be stored out of scope of the function and  referred to later